.ci/pytorch/windows/internal/xpu_install.bat

```diff
@@ -54,6 +57,7 @@ if not [%XPU_VERSION%]==[] if [%XPU_VERSION%]==[2025.0] (
     set XPU_EXTRA_VERSION=2025.0.1+1226
     set XPU_EXTRA_INSTALLED=0
     set XPU_EXTRA_UNINSTALL=0
+    set XPU_ENABLE_KINETO=1
```

Comment:
we don't need to set this var in this xpu support packages installation script

Reply:
Only oneAPI 2025 shipped `PTI` SDK, so we can only turn on it when oneAPI version is 2025+.

Reply:
I mean we'd better let the build/test decide whether enable Kineto. Otherwise, it will let all xpu CICD scenario enable the Kenito build.

Reply:
Okay, I will remove it and move to further dependency PR.